app/services/report/pdf.rb

```diff
@@ -0,0 +1,17 @@
+module Report
+  module PDF
+    def self.run(headers, rows)
+      pdf = Prawn::Document.new(options)
+      pdf.table [headers] + rows
+
+      pdf.render
+    end
+
+    def self.options
+      {
+        page_size: 'A4',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.